src/java.base/unix/native/libjli/java_md.h

```diff
@@ -61,8 +67,31 @@ static jboolean GetJREPath(char *path, jint pathsize, jboolean speculative);
 
 #if defined(_AIX)
 #include "java_md_aix.h"
+
+#define ZLIBNX_PATH "/usr/opt/zlibNX/lib"
+
+#ifndef POWER_9
+#define POWER_9 0x20000 /* 9 class CPU */
+#endif
+
+#ifndef POWER_10
+#define POWER_10 0x40000 /* 10 class CPU */
 #endif
 
+#define power_9_andup() ((POWER_9  == _system_configuration.implementation) \
+                        || (POWER_10 == _system_configuration.implementation))
+
+#ifndef SC_NX_CAP
+#define SC_NX_CAP 60
+#endif
+
+#ifndef NX_GZIP_PRESENT
+#define NX_GZIP_PRESENT 0x00000001
+#endif
+
+#define power_nx_gzip() (0 != ((long)getsystemcfg(SC_NX_CAP) & NX_GZIP_PRESENT))
+#endif /* _AIX */
```

Comment:
While our style prefers this closing comment, please don't change this existing line from openjdk.

Reply:
Fixed in d18063e